resource/ui/ts/models/proto.ts

```diff
@@ -113,6 +155,32 @@ module Models {
             available_range_count: number;
         }
 
+        /**
+         * Status is the common interface shared by NodeStatus and StoreStatus.
+         */
+        export interface Status{
```

Comment:
Are we moving to a generated model?  Do we need to write the generator ourselves or did you find one that works? My searches were fruitless.

It just seems very strange to have such a backwards relationship.  I have no problem with status, but we should have a function that creates a status from a node status and a store status.